icons/trending-up-down.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "Alportan"
+  ],
+  "tags": [
+    "arrow",
+    "arrows",
+    "estimated"
+    "up"
+    "down"
```

Invalid JSON here, but since `up` and `down` are part of the name, there's no need to add them, we could however add some extra tags to match the use case:

```suggestion
    "arrows",
    "estimated",
    "indeterminate",
    "data fluctuation",
    "uncertain",
    "forecast",
    "variable",
    "prediction",
    "dynamic",
    "volatile"
```

---

Makes sense! 🙌